docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,578 @@
+| `SelectionHandleMarkerImageLeft`   | PropertyMap | Specifies the display image used for left selection handle marker.  |
+| `SelectionHandleMarkerImageRight`  | PropertyMap | Specifies the display image used for right selection handle marker.  |
+| `SelectionHighlightColor`          | Vector4     | Specifies the selection highlight color.     |
+| `DecorationBoundingBox`            | Rectangle   | Specifies the position of decoration such as handles and so on within the on-screen area. |
+| `InputMethodSettings`              | PropertyMap | Specifies the settings related to the System Input Method, Key, and Value.  |
+| `InputColor`                       | Vector4     | Specifies the color of the new input text.         |
+| `EnableMarkup`                     | Boolean     | Specifies whether to enable the markup string, to process text within the markup tags using DALi application.<br>**Note**: By default, the markup string is disabled.  |
+| `InputFontFamily`                  | String      | Specifies the font family of the new input text. |
+| `InputFontStyle`                   | PropertyMap | Specifies the font style of the new input text.  |
+| `InputPointSize`                   | Float       | Specifies the font size of the new input text in points. |
+| `Underline`                        | PropertyMap | Specifies the default underline parameters.        |
+| `InputUnderline`                   | String      | Specifies the underline parameters of the new input text. |
+| `Shadow`                           | PropertyMap | Specifies the default shadow parameters.            |
+| `InputShadow`                      | String      | Specifies the shadow parameters of the new input text. |
+| `Emboss`                           | String      | Specifies the default emboss parameters         |
+| `InputEmboss`                      | String      | Specifies the emboss parameters of the new input text. |
+| `Outline`                          | PropertyMap | Specifies the default outline parameters.          |
+| `InputOutline`                     | String      | Specifies the  outline parameters of the new input text. |
+| `HiddenInputSettings`              | PropertyMap | Specifies hiding the input characters and shows default character for password or pin entry. |
+| `PixelSize`                        | Float       | Specifies the size of font in pixels.              |
+| `EnableSelection`                  | Boolean     | Specifies whether to enable or disable text selection.                   |
+| `Placeholder`                      | PropertyMap | Specifies the placeholder for text, color, font family, font style, point size, and pixel size. |
+| `Ellipsis`                         | Boolean     | Specifies whether to enable or disable ellipsis, if required.  |
+| `TranslatablePlaceholderText`      | String      | Specifies the   TranslatablePlaceholderText property that sets the SID value.  |

```

Comment:
@JoogabYun 
Kindly remove extra space
| `TranslatablePlaceholderText`      | String      | Specifies the TranslatablePlaceholderText property that sets the SID value.  |